core/modules/system/system.theme.inc

```diff
@@ -178,6 +180,9 @@ function theme_system_modules_fieldset($variables) {
     if ($required_by) {
       $requirements .= '<div class="admin-requirements">' . t('Required by:') . ' ' . implode(', ', $module['#required_by']) . '</div>';
     }
+    if ($modules_data[$key]->info['project'] !== 'backdrop') {
+      $requirements .= '<div class="admin-requirements">' . t('Links:') . ' ' . l(t("module's detail page"), 'https://backdropcms.org/project/' . $key) . ', ' . l(t("module's repository"), 'https://github.com/backdrop-contrib/' . $key) . '</div>';
```

Comment:
Can we please add another class to this `div` (something like `project-links`), and also tweak the link text:

```suggestion
      $requirements .= '<div class="admin-requirements project-links">' . t('Links:') . ' ' . l(t("project information"), 'https://backdropcms.org/project/' . $key) . ', ' . l(t("GitHub repository"), 'https://github.com/backdrop-contrib/' . $key) . '</div>';
```

Reply:
ok